### PR TITLE
Update news

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

**Review by borkmann — APPROVED** [review]
lgtm

**Review by manalibhutiyani — APPROVED** [review]

File: NEWS.rst [hunk 1]
```diff
@@ -2,9 +2,116 @@
...
+----------
+* Added support for Custom Resource Definition (CRD). Be aware that parallel
+  usage of CRD and Third party Resources (TPR) leads to unexpected behaviour.
+  See cilium.link/migrate-tpr for more details. Upgrade your
+  CiliumNetworkPolicy resources to cilium.io/v2 in order to use CRD. Keep them
+  at cilium.io/v1 to stay on TPR. (1169_, 1219_)
+* The CiliumNetworkPolicy resource now has a status field which contains the
+  status of each node enforcing the policy (1354_)
+* Added RBAC rules for v1/NetworkPolicy (1188_)
+* Upgraded to kubeadm 1.7.0 (1179_)
```
Comment (aanm) [inline NEWS.rst]: Should this be in CI? Also the link to #1179 doesn't seem to work?

Comment (manalibhutiyani) [reply NEWS.rst]: I am not able to click on any of the links actually.

Comment (aanm) [reply NEWS.rst]: @manalibhutiyani here https://github.com/cilium/cilium/blob/d45795fd5e9e675cc1b4701d3e2d7c95436c7afc/NEWS.rst

Comment (manalibhutiyani) [reply NEWS.rst]: thanks!

Comment (tgraf) [reply NEWS.rst]: Fixed

**Pull request merged**